packages/storybook/package.json

```diff
@@ -10,16 +12,18 @@
   "private": true,
   "repository": {
     "type": "git+ssh",
-    "url": "git@github.com:nl-design-system/lux.git"
+    "url": "git+ssh://git@github.com/nl-design-system/lux.git"
   },
   "scripts": {
     "build": "storybook build --output-dir dist/ --config-dir config/",
     "clean": "rimraf dist/",
     "lint-build": "tsc --noEmit --project tsconfig.json",
     "storybook": "storybook dev --config-dir config/ --port 6006",
-    "build-storybook": "storybook build --config-dir config/ --output-dir dist/"
+    "build-storybook": "storybook build --config-dir config/ --output-dir dist/",
+    "chromatic": "npx chromatic --project-token=chpt_f32f454ec222a1c"
```

Comment:
In scripts hoef je geen `npx` te gebruiken. Het volgende is hier voldoende:
```json
{
    "chromatic":  "chromatic"
}
```
Ik heb het token voor de zekerheid ook gereset. Het hoort niet onder versie beheer. Als je dit script wilt draaien kun je dat doen met:

```shell
$ CHROMATIC_PROJECT_TOKEN=<token> pnpm run chromatic
```

Of als je het meerdere malen binnen een terminal sessie wilt doen

```shell
$ export CHROMATIC_PROJECT_TOKEN=<token> # dit doe je eenmalig per terminal sessie
$ pnpm run chromatic
```

Reply:
Thanks